src/bug-filing/common/bug-filing-url-string-utils.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { EnvironmentInfo } from '../../common/environment-info-provider';
+import { CreateIssueDetailsTextData } from './../../common/types/create-issue-details-text-data';
+
+export class BugFilingUrlStringUtils {
+    public static footer(environmentInfo: EnvironmentInfo): string {
+        return (
+            'This accessibility issue was found using Accessibility Insights for Web ' +
```

Comment:
instead of 'Accessibility Insights for Web' we should use `title` from `src\content\strings\application.ts`